api/project/controllers/project.js

```diff
@@ -6,6 +6,7 @@
  */
 const { sanitizeEntity } = require('strapi-utils');
 
+const TEAM_EMAIL = 'team@devlaunchers.com';
```

Comment:
Let's not hard code this, we'll change this to another email.

Reply:
sounds good

Reply:
This is marked resolved but I still see it being hard coded.

Reply:
Oh I probably haven't committed. I realized you made an env var that has the subject which is the email, so I'm using that instead.